### PR TITLE
Added sorting of the master redirect file.

### PR DESCRIPTION
**Pull request by IEvangelist**
Hi @meganbradley,

Added the ability to sort the `.openpublishing.redirection.json` file. Open the command pallet and quickly sort the entire file alphabetically.

sort-redirection

### CODE REVIEW

**Comment by meganbradley** [comment]
@lamebrain FYI.

File: docs-markdown/src/controllers/master-redirect-controller.ts [hunk 1]
```diff
@@ -62,6 +72,33 @@ function showStatusMessage(message: string) {
...
+                redirects.redirections.sort((a, b) => {
+                    return naturalLanguageCompare(a.source_path, b.source_path);
+                });
+
+                fs.writeFileSync(
+                    file,
+                    JSON.stringify(redirects, ["redirections", "source_path", "redirect_url", "redirect_document_id"], 4));
+            }
+        }
+    }
```
Comment (lamebrain) [inline docs-markdown/src/controllers/master-redirect-controller.ts]: If `redirectFileName` does not exist, can we raise a notification?

Comment (IEvangelist) [reply docs-markdown/src/controllers/master-redirect-controller.ts]: Like a `postWarning`?

Comment (IEvangelist) [reply docs-markdown/src/controllers/master-redirect-controller.ts]: Done

**Pull request merged**